tests/testthat/test-readwritemagpie.R

```diff
@@ -78,23 +78,27 @@ test_that("read supports older formats", {
 })
 
 test_that("handling of spatial data works", {
-  td <- tempdir()
+  td <- withr::local_tempdir()
+
   md <- magclass:::magclassdata$half_deg
-  m05 <- new.magpie(paste0(md$region, ".", seq_len(dim(md)[1])), years = c(2000, 2001), fill = c(md$lon, md$lat))
+  m05 <- new.magpie(paste0(md$region, ".", seq_len(dim(md)[1])), years = c(2000, 2001),
+                    names = "nctest", fill = c(md$lon, md$lat))
   m10 <- mbind(m05, m05)
-  getNames(m10) <- c("bla", "blub")
-  expect_error(write.magpie(m10, file.path(td, "test.grd")), "no support for multiple variables")
-  expect_no_warning(write.magpie(m10, file.path(td, "test.nc")))
```

Comment:
why has this test been removed?

Reply:
It was just moved further down, it's still there. I found it clearer this way, because all tests regarding m05 now come first, and then the test on m10